nucypher/cli/painting.py

```diff
@@ -100,6 +103,13 @@ def paint_node_status(ursula, start_time):
              'Work Orders ......... {}'.format(len(ursula._work_orders)),
              teacher]
 
+    if not ursula.federated_only and ursula.stakes:
+        total_staked = f'Total Staked ........ {ursula.total_staked} NU-wei'
```

Comment:
How does anyone like format of `{ursula.total_staked:.2e} NU-wei`?

It would look something like `1.5e+21 NU-wei`

Reply:
How about  something that looks like `15000e+18 NU-Wei` ? Perhaps commas are more useful? `15,000|000,000,000,000,000,000`. Ideally this is something that can be useful for human verification.

Reply:
Yeah, that's some interesting formatting. `1.5e+21` is just very, very easy to get, and what you propose needs a custom formatter

Reply:
I like the [E-notation](https://en.wikipedia.org/wiki/Scientific_notation#E-notation) suggested by @michwill

Reply:
Will handle in PR #830 :heavy_check_mark: